Introduce custom metrics (#1003)
kubearchive/kubearchive: docs/modules/ROOT/pages/configuration/observability.adoc

= Observability on KubeArchive

> KubeArchive is instrumented to emit observability data: metrics,

Reviewer: Passive tense:

```suggestion
OpenTelemetry instruments KubeArchive to emit observability data: metrics,
```

Author: Thanks for keeping an eye on this. Fixed.

> traces and logs. This can be enabled and configured with environment
> variables present on the KubeArchive's components.

Reviewer: Passive tense:

```suggestion
traces and logs. Enable and configure the instrumentation setting the environment
variables present on the KubeArchive's components.
```
